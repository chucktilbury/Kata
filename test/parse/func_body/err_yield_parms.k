;-------------------------------------------------------------------------------
; yield requires a compound reference
;-------------------------------------------------------------------------------

func name()() {

	for(symbol in expression) {
		yield(123)
	}
}
