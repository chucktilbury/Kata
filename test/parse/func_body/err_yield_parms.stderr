Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_yield_parms.k: 8: 12: expected a compound reference but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_yield_parms.k: 11: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_yield_parms.k: 11: 1: expected function body but got end of file
